subcommand/flags/http.go

```diff
@@ -0,0 +1,144 @@
+package flags
+
+import (
+	"flag"
+	"io/ioutil"
+	"strings"
+
+	"github.com/hashicorp/consul/api"
+)
+
+// Taken from https://github.com/hashicorp/consul/blob/master/command/flags/http.go
+// with flags we don't use removed. This was done so we don't depend on internal
+// Consul implementation.
+
+// HTTPFlags are flags used to configure communication with a Consul agent.
+type HTTPFlags struct {
+	address       StringValue
+	token         StringValue
+	tokenFile     StringValue
+	caFile        StringValue
+	caPath        StringValue
+	tlsServerName StringValue
```

Comment:
Should we keep client cert and client key? If someone is using this binary standalone they may be passing these to the commands because they've set up the client to accept mTLS connections.

Reply:
right, added back in